docs/DeveloperGuide.adoc

```diff
@@ -349,6 +383,7 @@ _{More to be added}_
 .  Should work on any <<mainstream-os,mainstream OS>> as long as it has Java `11` or above installed.
 .  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
 .  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+.  Should handle text input of english alphabets.
 
```

Comment:
Added some NFRs related to performance, scalability and usability! Perhaps you can consider adding other constraints given by the tP project guidelines as well such as the single user constraint.